config/default.yml

```diff
@@ -1,3 +1,10 @@
+Ezcater/GraphqlFieldsNaming:
+  EnforcedStyle: snake_case
+  Enabled: true
+  SupportedStyles:
+    - snake_case
+    - camelCase
+
```

Comment:
Do you want to restrict this cop to only apply to certain directories, e.g. `app/graphql`?

Reply:
Good call! just updated.